### PR TITLE
chore: add missing migration changes

### PR DESCRIPTION
**Pull request by erik-slovak**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,9 @@
   "dependencies": {
     "@txo/config-manager": "^3.1.0"
   },
+  "peerDependencies": {
+    "redux-saga": "^1.2.1"
```
Comment (rostislav-simonik) [inline package.json:40]: @erik-slovak why is it peer dependency?

Comment (erik-slovak) [reply package.json]: I added that by mistake, removed

File: package.json [hunk 2]
```diff
@@ -43,39 +43,42 @@
...
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "eslint-plugin-redux-saga": "^1.3.2",
     "husky": "^8.0.1",
     "immutability-helper": "^3.1.1",
     "jest": "^29.1.2",
     "lint-staged": "^13.0.3",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
```
Comment (rostislav-simonik) [inline package.json:70]: why this is needed ?

File: package.json [hunk 3]
```diff
@@ -43,39 +43,42 @@
...
+    "redux-saga": "^1.2.1",
     "rimraf": "^3.0.2",
     "semantic-release": "^19.0.5",
     "semantic-release-slack-bot": "^3.5.3",
     "ts-jest": "^29.0.3",
     "typescript": "^4.8.4"
   },
+  "resolutions": {
+    "@types/react": "^17.0.0"
+  },
```
Comment (rostislav-simonik) [inline package.json:81]: why thiis needed ?

**Review by rostislav-simonik — APPROVED** [review]

**Pull request merged**